src/Avalonia.Themes.Fluent/Controls/ComboBox.xaml

```diff
@@ -159,10 +153,9 @@
     <Setter Property="IsVisible" Value="False" />
   </Style>
 
-  <Style Selector="ComboBox /template/ Path#DropDownGlyph">
-    <Setter Property="Fill" Value="{DynamicResource ComboBoxDropDownGlyphForeground}" />
+  <Style Selector="ComboBox /template/ PathIcon#DropDownGlyph">
+    <Setter Property="Foreground" Value="{DynamicResource ComboBoxDropDownGlyphForeground}" />
```

Comment:
Strangely this does not work. I can change Data property here just fine but the `Foreground` property is ignored completely. I can set it to `Red` here and there will be no effect.

If the `Foreground` is set above in the `PathIcon` declaration it works fine. If `Foreground` is set within the template of `PathIcon` itself it works fine. I can't imagine what priority-type issue is causing the issue here...

Reply:
Addressed #7859